tgstation.dme

```diff
@@ -4923,13 +4922,15 @@
 #include "voidcrew\mapping\shuttles\pill.dm"
 #include "voidcrew\mapping\shuttles\pill_black.dm"
 #include "voidcrew\mapping\shuttles\pubby.dm"
-#include "voidcrew\mapping\shuttles\spitfire.dm"
 #include "voidcrew\mapping\shuttles\nanotrasen\bead.dm"
 #include "voidcrew\mapping\shuttles\nanotrasen\phalanx.dm"
 #include "voidcrew\mapping\shuttles\nanotrasen\thunderbird.dm"
 #include "voidcrew\mapping\shuttles\syndicate\blackbeard.dm"
+#include "voidcrew\mapping\shuttles\syndicate\boyardee_b.dm"
 #include "voidcrew\mapping\shuttles\syndicate\geneva.dm"
 #include "voidcrew\mapping\shuttles\syndicate\hyena.dm"
+#include "voidcrew\mapping\shuttles\syndicate\luxembourg.dm"
+#include "voidcrew\mapping\shuttles\syndicate\spitfire.dm"
```

Comment:
I think originally I wanted to limit NT/Syndicate ships to 3 each, now there's 3 NT and 6 Syndicate

Reply:
That's actually 100% fair - I'll repath to what they were before as the alternative is an arms race between the two, mapping wise - and although that'd be fitting, scope creep bad ackshually.

That being said, 3 of each means they've gotta have their best feet forward, so I guess I have my next remap targets.